pypesto/objective/function.py

```diff
@@ -114,6 +114,17 @@ def has_res(self) -> bool:
     def has_sres(self) -> bool:
         return callable(self.sres) or self.sres is True
 
+    def get_config(self):
+        info = {}
```

Comment:
it makes sense to always start by `config = super().get_config()` and then extend it.